File: C++/Step02-If/10817.cpp
//
//  10817.cpp
//  Step2-If
//
//  Created by 김예빈 on 2019. 8. 28..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    int a, b, c;
    
    cin >> a >> b >> c;
    
    if (a >= b && a >= c)
    {
        if (b > c)
            cout << b;
        else
            cout << c;
    } else if (b >= a && b >= c)
    {
        if (a > c)
            cout << a;
        else
            cout << c;
    } else /*if (c >= a && c >= b)*/
    {
        if (a > b)
            cout << a;
        else
            cout << b;
    }
    
    return 0;
}

File: BaekJoon Online Judge/Swift/etc/10817.swift
//
//  10817.swift
//  Step02-If
//
//  Created by Yebin Kim on 2019/10/28.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

var input = readLine()

if let input = input {
    let arr = input.components(separatedBy: " ")
    
    let num1 = Int(arr[0])!
    let num2 = Int(arr[1])!
    let num3 = Int(arr[2])!
    
    if(num1 >= num2 && num1 >= num3) {
        if(num2 > num3) {
            print(num2)
        } else {
            print(num3)
        }
    } else if(num2 >= num1 && num2 >= num3) {
        if(num1 > num3) {
            print(num1)
        } else {
            print(num3)
        }
    } else {
        if(num1 > num2) {
            print(num1)
        } else {
            print(num2)
        }
    }
}
